src/main/kotlin/no/nav/familie/ef/søknad/integration/SøknadClient.kt

```diff
@@ -25,43 +24,52 @@ import java.util.UUID
 
 @Service
 class SøknadClient(private val config: MottakConfig,
+                   private val featureToggleService: FeatureToggleService,
                    @Qualifier("tokenExchange") operations: RestOperations)
     : AbstractPingableRestClient(operations, "søknad.innsending") {
 
     override val pingUri: URI = config.pingUri
 
+    fun config(): MottakConfig {
+        return if (featureToggleService.isEnabled("familie.ef.soknad.api.gcp")) {
```

Comment:
Lurer på om vi skulle ha en toggle for å skru av den helt sånn att vi kan skru av den helt mens vi migrerer i prod for å ikke få inn data noe sted i noen sekunder. 
eks att den kaster feil hvis den toggelen er slått på

Reply:
Ok, jeg la til at det i config()-metoden. Går ut ifra at pod vil stå som running og ikke får evig crashloop